services/github/src/github-auth.js

```diff
@@ -22,50 +22,31 @@ const retryPlugin = (octokit, options) => {
     }
   });
 };
-const Github = require('@octokit/rest').plugin([retryPlugin]);
+const Octokit = require('@octokit/rest').plugin([retryPlugin]);
 
 module.exports = async ({cfg}) => {
-  let github = new Github({
-    promise: Promise,
-  });
-
-  let setupToken = _ => {
-    let inteToken = jwt.sign(
-      {iss: cfg.github.credentials.integrationId},
+  const getAppGithub = async () => {
+    const inteToken = jwt.sign(
+      {iss: cfg.github.credentials.appId},
       cfg.github.credentials.privatePEM,
       {algorithm: 'RS256', expiresIn: '1m'},
     );
-    try {
-      github.authenticate({type: 'app', token: inteToken});
-    } catch (e) {
-      debug('Authentication as app failed!');
-      throw e;
-    }
-    return github;
+
+    return new Octokit({auth: `bearer ${inteToken}`});
+  };
+
+  const getInstallationGithub = async (inst_id) => {
+    const inteGithub = await getAppGithub();
```

Comment:
Let's not mutate global variables, please 👍

Lovely refactor of this module in general